kuma/users/adapters.py

```diff
@@ -247,6 +251,14 @@ def save_user(self, request, sociallogin, form=None):
             # multiple associated Persona social accounts (each identified
             # by a unique email address).
             user.socialaccount_set.filter(provider='persona').delete()
+            # Send an event to Google Analytics that the authentication
+            # from one provider could be added to an account created by a
+            # *different* provider (originally).
+            track_event(
+                CATEGORY_SIGNUP_FLOW,
+                ACTION_SOCIAL_AUTH_ADD,
+                f'{sociallogin.account.provider}-added'
+            )
```

Comment:
We need to add something like the following after this `track_event()` call:
```python
request.social_auth_added = True
```
which we'll check for when handling the `user_signed_up` and `user_logged_in` signals (see further comments below).